snap/local/runtime-helpers/bin/kong-postgres-setup.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/sh
+
+# this script is called from the install hook to set up postgres
+# and also from the post-refresh hook. It handles these three scenerios:
+# - new install: create config file and database
+# - refresh from 2.0/stable or older - a snap using postgres 10
+# - refresh from a 2.1/stable or newer snap using postgres 12
```

Comment:
Please use versions instead of channels, otherwise this comment would quickly become outdated. The 2.1/stable channel currently points to a version which uses postgres 10 but that may change in future. We also have other channels which the given scenarios apply to: latest/stable, latest/beta, 2.x/beta, 2.1/edge.

Perhaps it is better to leave the channels and versions out of this comment and just mention the postgres versions. We'll have to use epochs to control which channels gets this update and that is beyond the scope of this script.